### PR TITLE
Change default param values

### PR DESCRIPTION
**Pull request by jdebacker**
This PR updates values of some parameters in `default_parameters.json` to be more consistent with our preferred specification of OG-USA.

### CODE REVIEW

**Comment by jdebacker** [comment]
@rickecon I've had all tests pass locally -- this is ready for your review.  Thanks!

**Comment by jdebacker** [comment]
@rickecon Are you ok with these changes to the default parameter values?  I plan to merge this later today unless you object.

File: cs-config/cs_config/functions.py [hunk 1]
```diff
@@ -163,9 +163,7 @@ def run_model(meta_param_dict, adjustment):
     # Solve baseline model
     base_spec = {
         **{'start_year': meta_param_dict['year'],
-        'debt_ratio_ss': 2.0, 'r_gov_scale': 1.0, 'r_gov_shift': 0.02,
-        'zeta_D': [0.4], 'zeta_K': [0.1], 'initial_debt_ratio': 0.78,
-        'initial_foreign_debt_ratio': 0.4, 'tax_func_type': 'linear',
+        'tax_func_type': 'linear',
```
Comment (rickecon) [inline cs-config/cs_config/functions.py]: @jdebacker. Why this change in the baseline specification to `'tax_func_type': 'linear',`?

Comment (jdebacker) [reply cs-config/cs_config/functions.py]: This is for C/S so that things run faster.  Linear tax functions take about 15 minutes, whereas the `DEP` functions take a couple hours to estimate.   We are also doing `age_dependent=False` to save computational time for now.

Comment (rickecon) [reply cs-config/cs_config/functions.py]: OK
